db/config.cc

```diff
@@ -1004,7 +1004,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , enable_sstables_md_format(this, "enable_sstables_md_format", value_status::Unused, true, "Enable SSTables 'md' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
     , sstable_format(this, "sstable_format", value_status::Used, "me", "Default sstable file format", {"md", "me"})
     , uuid_sstable_identifiers_enabled(this,
-            "uuid_sstable_identifiers_enabled", liveness::LiveUpdate, value_status::Used, true, "If set to true, each newly created sstable will have a UUID "
+            "uuid_sstable_identifiers_enabled", value_status::Unused, true, "If set to true, each newly created sstable will have a UUID "
```

Comment:
Do we need to also change the message?
(like adding "deprecated" or alike, or it is for free when marked `value_status::Unused`)

Reply:
Currently, "Unused" options are not displayed in the `--help` output or documented. see https://github.com/scylladb/scylladb/blob/2148e33d37b952158b592cd475110910681375f7/utils/config_file.hh#L72-L81 and https://opensource.docs.scylladb.com/master/reference/configuration-parameters.html